docs/graphs/index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: Graph Edge
+title: Graphs
 sidebar_position: 1
```

Comment:
Use two-digit numbers for sidebar positions (10, 20, 30). This makes it easier to insert files later, less renumbering.